src/aragonjs-wrapper.js

```diff
@@ -350,8 +350,8 @@ const initWrapper = async (
       forceLocalStorage: workerFrameSandboxDisabled,
     },
     events: {
-      // Infura hack: delay event processing for 1s
-      subscriptionEventDelay: 1000,
+      // Infura hack: delay event processing for 5s
+      subscriptionEventDelay: 5000,
```

Comment:
Wait, didn't I see a merge earlier today removing this and saying it is not the correct way?
cc @sohkai

Reply:
That would be to remove the hotfix once we stop using Infura :).